### PR TITLE
build(aio): lint examples

### PR DESCRIPTION
**Pull request by Foxandxss**
I added the tslint setup we had in the old site just for the examples. I added it to `yarn build` so travis will run it too.

### CODE REVIEW

File: aio/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "ng": "yarn check-env && ng",
     "start": "yarn check-env && ng serve",
-    "build": "yarn check-env && yarn setup && ng build -prod -sm",
+    "build": "yarn check-env && yarn setup && example-lint && ng build -prod -sm",
```
Comment (petebacondarwin) [inline aio/package.json]: Why add the example lint here?
I would expect it to be part of the `yarn lint` command?

Comment (Foxandxss) [inline aio/package.json]: you are super right

**Review by petebacondarwin — APPROVED** [review]
LINTING!!!

File: aio/package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "ng": "yarn check-env && ng",
     "start": "yarn check-env && ng serve",
     "build": "yarn check-env && yarn setup && ng build -prod -sm",
-    "lint": "yarn check-env && yarn docs-lint && ng lint",
+    "lint": "yarn check-env && yarn docs-lint && ng lint && example-lint",
```
Comment (gkalpak) [inline aio/package.json]: _example-lint_ --> _yarn example-lint_

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: aio/package.json (hunk 2 above)
Comment (petebacondarwin) [inline aio/package.json]: oops!

**Review by gkalpak — APPROVED** [review]

**Pull request merged**